test/models/image/images_tests.rb

```diff
@@ -2,6 +2,12 @@
 describe "Fog::Image[:openstack] | images" do
   before do
     @instance = Fog::Image[:openstack].create_image(:name => "model test image").body
+    @private = Fog::Image[:openstack].create_image(
```

Comment:
This falls back in testing the `Fog::Image::OpenStack::V1::Image` model and therefore fails.

I suggest splitting up the test into v1 and v2 and call those explicitly, e.g. `Fog::Image::OpenStack::V2.new.create_image`